test/run_tests.sh

```diff
@@ -109,7 +109,8 @@ function run_op_tests {
   run_dynamic python3 "$CDIR/../../test/nn/test_convolution.py" "$@" -v TestConvolutionNNDeviceTypeXLA
   run_dynamic python3 "$CDIR/../../test/test_type_promotion.py" "$@" -v TestTypePromotionXLA
   run_dynamic python3 "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
-  run_dynamic python3 "$CDIR/test_dynamic_shapes.py" "$@"
+  run_dynamic python3 "$CDIR/test_dynamic_shapes.py" "$@" --verbosity=$VERBOSITY
```

Comment:
I don't think you need --verbosity=$VERBOSITY unless you want to add
https://github.com/pytorch/xla/blob/6e6bb07e696c51a555a7d33433508ba236703a35/test/test_operations.py#L13

and https://github.com/pytorch/xla/blob/6e6bb07e696c51a555a7d33433508ba236703a35/test/test_operations.py#L2277

Reply:
Got it. Thanks. Removed the flag.